### PR TITLE
Resolve breakpoint target files statically

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL.

File: lib/debuglet.js [hunk 1]
```diff
@@ -83,42 +84,44 @@ function Debuglet(config, logger) {
...
-      that.logger_.error('Unable to compute unique id of the application', err);
-      if (opt_callback) {
-        opt_callback(err);
-      }
-      return;
-    }
-    that.logger_.info('Unique ID for this Application: ' + uid);
-
-    that.debugletApi_.init(uid, function(err) {
+  scanner.scan(that.config_.workingDirectory, function(err, hash, fileStats) {
```
Comment (ofrobots) [inline lib/debuglet.js]: Previous we would do a scan only if `workingDirectory` contained a `package.json`. I think this is important. This prevents us from doing a large scan of the whole file-system accidentally if the node process is started in a weird location. Could you bring back that behaviour?

Comment (ofrobots) [inline lib/debuglet.js]: Maybe you can bring that logic into this file and refuse to enable the debulet unless we can find a package.json in `config.workingDirectory`.

Comment (matthewloring) [inline lib/debuglet.js]: This logic has been moved and the uid file has been deleted.

**Comment by matthewloring** [comment]
@ofrobots Ready for another look

File: test/test-uid.js [hunk 2]
```diff
@@ -1,51 +0,0 @@
-/**
```
Comment (ofrobots) [inline test/test-uid.js]: Can you keep these tests, perhaps move them to test-agent.js?

Comment (ofrobots) [inline test/test-uid.js]: err... test-debuglet.js

Comment (matthewloring) [inline test/test-uid.js]: Yup, they've been reintroduced.

**Comment by ofrobots** [comment]
LGTM.

**Pull request merged**